src/ingest_validation_tools/directory-schemas/codex.yaml

```diff
@@ -1,14 +1,7 @@
-# A dataset consists of a single CODEX experimental run, and is
-# collected by the Akoya software into a single directory.
-#
-# directory beginning with drv_ contains analyzed files produced.
-# There are a few files in this directory that are used for the HuBMAP
-# data processing pipelines (channelNames.txt, experiment.json, segmentation.json)
-#
-# directory beginning with src_ contains the raw (unprocessed by Akoya) files.
+# A dataset consists of a single CODEX experimental run
 
 -
-  pattern: '[^/]*NAV[^/]*\.tif'
+  pattern: '[^/]NAV[^/]*\.tif'
```

Comment:
This means there is always one character before `NAV`... is that correct? 
Will pass:
```
1NAV-abc.tif
-NAV.tif
```
Will fail:
```
NAV-abc.tif
123NAV.tif 
```

Reply:
@mccalluc when you are right, you are right

![Screen Shot 2021-07-21 at 4 56 17 PM](https://user-images.githubusercontent.com/413752/126559016-9d6e2447-0262-476c-9e6f-249c32733e20.png)